Analysis/collinearity_analysis.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+from scipy.stats import pearsonr
+import matplotlib.pyplot as plt
+import seaborn as sns
+import fire
+def main():
+    dataset_path = './data/static_analysis_data/dataset.csv'
+    dataset = pd.read_csv(dataset_path)
+
+    funcs_number = dataset['#funcs']
+    unit_test_funcs_number = dataset['#unit']
+    proptery_based_test_funcs_number = dataset['#proptery_based']
+    fuzz_target_number = dataset['#fuzz_target']
+
+
+    correlation_unit_funcs = pearsonr(funcs_number, unit_test_funcs_number)[0]
+    correlation_proptery_based_funcs = pearsonr(funcs_number, proptery_based_test_funcs_number)[0]
+    correlation_fuzz_target_funcs = pearsonr(funcs_number, fuzz_target_number)[0]
+
+    correlations = [correlation_unit_funcs,correlation_proptery_based_funcs,correlation_fuzz_target_funcs]
+    predictors = ["unit test functions number","property-based test functions number","fuzz target number"]
```

Comment:
names too long, making x-axis on the figure hard to read

Reply:
OK， I will change it

> names too long, making x-axis on the figure hard to read